examples/graphql-client-error-handling/graphql_client_error_handling.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/graphql;
+import ballerina/io;
+
+type ProfileResponse record {|
+    *graphql:GenericResponseWithErrors;
+    record {|Person profile;|} data;
+|};
+
+type Person record {|
```

Comment:
Let's use `Profile` as type name in all the places.

```suggestion
type Profile record {|
```

Reply:
Updated